src/main/java/org/zalando/nakadi/security/Client.java

```diff
@@ -1,22 +1,29 @@
 package org.zalando.nakadi.security;
 
-public interface Client {
+import org.zalando.nakadi.exceptions.IllegalScopeException;
 
-    Client PERMIT_ALL = clientId -> true;
+import java.util.Set;
 
-    boolean is(String clientId);
+public class Client implements IClient {
```

Comment:
He is authenticated because we have checked and validated OAuth token. He is just not authorized yet do do some things.